addon/package.json

```
@@ -86,7 +86,7 @@
   "peerDependencies": {
     "ember-source": ">=3.28.0",
     "qunit": "^2.0.0",
-    "sinon": "^15.0.3 || ^16.0.0 || ^17.0.0"
+    "sinon": "^15.0.3 || ^16.0.0 || ^17.0.0 || ^18.0.0"
```

What do you think about just changing this to be future-inclusive until we run into a compatibility issue down the road, so we don't have to keep updating it?
```suggestion
    "sinon": ">=15.0.3"
```